prometheus/alertmanager.yml

```diff
@@ -0,0 +1,32 @@
+global:
```

Comment:
Alertmanager configs added, but no description and guide in README how to test alerts

Reply:
added in readme